Tests/CacheTests.swift

```diff
@@ -83,8 +83,13 @@ class CacheTestCase: BaseTestCase {
 
         urlCache = {
             let capacity = 50 * 1024 * 1024 // MBs
-            #if targetEnvironment(macCatalyst)
-            return URLCache(memoryCapacity: capacity, diskCapacity: capacity)
+            // swiftformat:disable indent
+            #if swift(>=5.1) && !os(Linux)
```

Comment:
I think the `targetEnvironment` check should work here.

Reply:
Looks like this also broke the Catalyst build in general.

Reply:
Fixed!